df.graphics.xml

```diff
@@ -92,6 +92,7 @@
     </enum-type>
 
     <class-type type-name='enabler' original-name='enablerst' custom-methods='true'>
+        <extra-include filename='Hooks.h'/>
```

Comment:
Similar to my Bitfield.pm comment, referring to a DFHack header in an XML file feels out of place to me. Are there other options?